doc/python/conf.py.in

```diff
@@ -0,0 +1,189 @@
+# Copyright (c) 2016, EPFL/Blue Brain Project
```

Comment:
No, it's not generated automatically. It's the example Sphinx configuration file, the commented code is there for exactly the same reasons we keep the commented options in Doxygen configuration files (whatever are the reasons).